CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,58 @@
+# Changelog 2.3
+
+## 2.3.0 - 2025-02-21
+
+### Added
+
+- Add `Proxy` mode for Flow Aggregator to send flows directly without buffering or aggregation. ([#6920](https://github.com/antrea-io/antrea/pull/6920) [#6961](https://github.com/antrea-io/antrea/pull/6961), [@antoninbas])
+- Support version skew between Antrea Agent and Flow Aggregator to improve upgrade robustness. ([#6912](https://github.com/antrea-io/antrea/pull/6912), [@antoninbas])
+- Add `clusterId` to aggregated records for Flow Aggregator. ([#6769](https://github.com/antrea-io/antrea/pull/6769), [@antoninbas])
+- Add `checksum/config` annotation to the Deployment of Flow Aggregator. ([#6967](https://github.com/antrea-io/antrea/pull/6967), [@antoninbas])
+- Add more fields in `BGPPolicy` API to support BGP confederation. ([#6905](https://github.com/antrea-io/antrea/pull/6905), [@hongliangl])
```

Comment:
I wonder if we should omit this changelog entry given that we have postponed implementation to v2.4
It may create some confusion

Reply:
Removed it, I will include it in 2.4 release log.